fcc-test-suite-template-master(1)/babel/index.babel

```diff
@@ -0,0 +1,27 @@
+// !! IMPORTANT README:
```

Comment:
- [ ] Kindly use a descriptive readme file that describes what the project does, how to set it up, links to your social platforms etc. This would help other people viewing your repository understand your project better. You could refer to the [https://github.com/microverseinc/readme-template](url) 🙌🏼👍🏼